lib/galaxy/tool_util/deps/brew_exts.py

```diff
@@ -51,7 +51,7 @@ class BrewContext:
     def __init__(self, args=None):
```

Comment:
this whole file should be axed... but next PR 😆. I think I removed the Planemo bits that used this.